docs/contribute/core.md

```diff
@@ -94,10 +94,10 @@ feat!: feature with breaking change
 
 The full list of `<types>` are: `feat`, `fix`, `docs`, `style`, `refactor`, `perf`, `test`, `build`, `ci`, `chore`, `revert`.
 
-* `!`: indicates a breaking change
-* `fix`: will create a new SemVer `patch`
-* `feat`: will create a new SemVer `minor`
-* `<type>!`: will create a new SemVer `major`
+- `!`: indicates a breaking change
+- `fix`: will create a new SemVer `patch`
+- `feat`: will create a new SemVer `minor`
+- `<type>!`: will create a new SemVer `major`
```

Comment:
Why were these changed?

Reply:
I am so sorry for that. It should be not changed. 
In order to distinguish which is changed by me and the `format` tool, I have made two commits.
We can see easily that this change is because of the `format` tool.